201-vm-linux-dynamic-data-disks/README.md

```diff
@@ -0,0 +1,5 @@
+# centos_dynadisk
```

Comment:
Rename to "Setup Linux Dynamic data disks" or something that explains a bit about the scenario